.changeset/famous-pans-press.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/account": minor
+---
+
+feat!: read malleable fields from transaction status on subscription
```

Comment:
Nit - in this release, we:
 - Reduced required roundtrips & bandwidth for submitting TXs